website/forms/php/filters/validation/fail-mimetype-validation-when-file-not-on-fs.md

```diff
@@ -3,4 +3,13 @@ id: fail-mimetype-validation-when-file-not-on-fs
 title: Fail mimetype validation
 ---
 
-*Coming soon*
+By default, mime-types are validated from the filesystem mimetype. However, in case the file is not present on the filesystem for any reason, this will fall back to the POST-provided mimetype.
+
+Using this filter, you can force Eightshift Forms to fail every file upload where it can't validate the mimetype from the filesystem.
+
+```php
+add_filter('es_forms_validation_force_mimetype_from_fs', function(): bool {
+	return true;
+})
```

Comment:
```suggestion
By default, MIME types are validated by the filesystem. However, if the file is not present on the filesystem for any reason, this will fall back to the POST-provided MIME types.

Using this filter, you can force Forms to fail every file upload where it can't validate the MIME type from the filesystem.

```php
add_filter('es_forms_validation_force_mimetype_from_fs', function(): bool {
	return true;
})
```